.github/workflows/build-package-docs.yaml

```diff
@@ -55,6 +55,16 @@ jobs:
   build-package-docs:
     runs-on: ubuntu-latest
     steps:
+    - name: Log the workflow inputs
+      run: |
```

Comment:
I don't honestly know when we'd ever run this workflow without the deployment checkbox. I've been doing it without deploy  for 'debug' purposes, and it's helpful to see what's there in my mind even if we don't deploy.

Reply:
In some cases, I think it could be useful to run a build against a PR then download the tarball and check things locally without the need to deploy. For some of the work on docs tooling (like antsibull-docs), what gets written to the logs can be more interesting than the rendered content.

I'd also like to start doing nightly builds (without deploy) that would help us identify issues a lot more quickly. Once we get the PR to notify of build failures merged, we can start running nightlies of the package docs.

Reply:
Another thing to consider is that making this step conditional, with `if: fromJSON(github.event.inputs.deploy)` , gives us an implicit statement of whether the "deploy" checkbox was selected.

If don't want to make this step conditional, I suggest we make that an explicit statement. Maybe that will give us consistency, if being a little verbose.

Reply:
Trying to clarify my last comment some more....

With `if: fromJSON(github.event.inputs.deploy)` the "Deployment details" section appears in the workflow summary only when the "deploy" checkbox is selected. Given the deployment details are relevant for approvers in the docs-community-maintainers group, we don't need to see them unless there is a deployment.

When we see the "Deployment details" section appears in the workflow summary, we know that the "deploy" checkbox was selected to run the deploy job.

However, if the "Deployment details" section appears in every workflow summary, then we should include a line in the deployment details to make it obvious if the deploy job ran. Otherwise we'll see values for the "Deployment details" section without knowing whether the deploy job was actually run. This seems slightly more verbose to me but maybe more consistent.

I think it comes down to one of the following:

`if: fromJSON(github.event.inputs.deploy)` - Add deployment details to the summary only if there is a deployment.

`echo "Deploy?: ${{ github.event.inputs.deploy }}" >> "${GITHUB_STEP_SUMMARY}"` - Always add deployment details to the summary and explicitly show whether the deploy job was run.

Reply:
chose for now to have deployment details only if deploy checked. thanks!